src/Sample.xoo

```diff
@@ -1,3 +1,4 @@
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:3000/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:3000/project/issues?id=sample&branch=feature1&issues=AWD99-lxVgAt5i3vVKWI&open=AWD99-lxVgAt5i3vVKWI)

[ISSUE_KEY]: AWD99-lxVgAt5i3vVKWI

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED